examples/tutorials_generator.ipynb

```diff
@@ -583,7 +574,7 @@
     {
      "output_type": "stream",
      "name": "stdout",
-     "text": "The url queries that are made for each nodes are:\n - for allergy node is http://hapi.fhir.org/baseR4/AllergyIntolerance?code=http%3A%2F%2Fsnomed.info%2Fsct%7C22749300\n - for patient node is http://hapi.fhir.org/baseR4/Patient?birthdate=1976\n - for practitioner node is http://hapi.fhir.org/baseR4/Practitioner?address-city=paris\n - for condition node is http://hapi.fhir.org/baseR4/Condition?code=http%3A%2F%2Fsnomed.info%2Fsct%7C386661006\n"
+     "text": "The url queries that are made for each nodes are:\n - for allergy node is http://hapi.fhir.org/baseR4/AllergyIntolerance?patient%3APatient.birthdate=1976&asserter%3APractitioner.address-city=paris\n - for patient node is http://hapi.fhir.org/baseR4/Patient?_has%3ACondition%3Asubject%3Acode=http%3A%2F%2Fsnomed.info%2Fsct%7C386661006&_has%3AAllergyIntolerance%3Apatient%3Acode=http%3A%2F%2Fsnomed.info%2Fsct%7C22749300\n - for practitioner node is http://hapi.fhir.org/baseR4/Practitioner?_has%3AAllergyIntolerance%3Aasserter%3Acode=http%3A%2F%2Fsnomed.info%2Fsct%7C22749300\n - for condition node is http://hapi.fhir.org/baseR4/Condition?subject%3APatient.birthdate=1976\n"
```

Comment:
I have the impression that the urls do not contain any more condition on the target resource of the request. For example, the query on the patient has no more conditions on his age, right?

Reply:
Yes indeed thank you, it will be changed in the next commit

Reply:
Great!